### PR TITLE
Fix django issue about is_anonymous django deprecation fixes #126

### LINKED ISSUE #126: Fix django issue about is_anonymous django deprecation
RemovedInDjango20Warning: Using user.is_authenticated() and user.is_anonymous() as a method is deprecated. Remove the parentheses to use it as an attribute.

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/env python
 import os
 import sys
+import warnings
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "dashboard.settings")
 
+    # Regex for which modules to ignore warnings from
```
Comment (zqian) [inline manage.py]: does this mean any warning from djangosaml2 will all be ignored?

Comment (jonespm) [reply manage.py]: Yeah, that should probably target the line specifically, let me look into that.

Comment (jonespm) [reply manage.py]: I modified this so it only ignores DeprecationWarning(s). I could target line number but that might change if the code changes.

Comment (zqian) [reply manage.py]: looks good!

**Pull request merged**